lib/hammer_cli_csv/csv.rb

```diff
@@ -3,9 +3,59 @@
+      def add_list(heading, items)
+        items.delete_if do |item|
+          if item.class == Clamp::Subcommand::Definition
+            subcommand_class = item.subcommand_class
+            subcommand_class == HammerCLICsv::CsvCommand::ActivationKeysCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ArchitecturesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ComputeProfilesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ComputeResourcesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ContainersCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ContentHostsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ContentViewsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ContentViewFiltersCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::DomainsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::HostCollectionsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::HostsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::HostGroupsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::InstallationMediasCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::JobTemplatesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::LifecycleEnvironmentsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::LocationsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::OperatingSystemsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::OrganizationsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::PartitionTablesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ProductsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ProvisioningTemplatesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::PuppetEnvironmentsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::PuppetFactsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::PuppetReportsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::ReportsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::RolesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::SmartProxiesCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::SpliceCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::SubnetsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::SubscriptionsCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::SyncPlansCommand ||
+                subcommand_class == HammerCLICsv::CsvCommand::UsersCommand
+          else
+            false
+          end
```

Comment:
Are newly added commands expected to work out of the box? If not, I would say a whitelist is better. If new stuff should "just work", then a blacklist works better.

Also, can this be changed to an array of class names? That way you could keep it up top in a constant and update as needed, then just check if `subcommand_class` exists in the array. I misread this block a couple of times and thought it was doing something totally different :evergreen_tree: